### PR TITLE
Fix entries of a deleted user break the UI

### PR DESCRIPTION
**Pull request by kerstenremco**
Removed user (owner object) isn't passed to EJS. Included fallback icon and text
Fix for #4159 #2041

### CODE REVIEW

**Comment by kerstenremco** [comment]
Hi @villcabo,

I can't reproduce this. I've tried to setup a user and a host, delete that user, but the hostslist is still rendering.
I even delete the whole userrecord in the database.

Can you provide steps to reproduce this?

SCR-20241125-osjd

**Comment by kerstenremco** [comment]
accidentally closed this PR, see #4187

**Comment by villcabo** [comment]
@kerstenremco Thanks it's working

File: frontend/js/app/nginx/access/list/item.ejs
Comment (peterlinkpr) [inline frontend/js/app/nginx/access/list/item.ejs:1]: Please, What is the exact location of this file?

**Pull request closed without merging**